sssom/cli.py

```diff
@@ -53,6 +53,7 @@
     compare_dataframes,
     dataframe_to_ptable,
     filter_redundant_rows,
+    flip_mappings,
```

Comment:
replace the entire `flip` terminology with `invert`

Reply:
i.e. `invert_mappings`

Reply:
Even the CLI command `flip` => `invert`?

Reply:
yes lets go for unified termonology. Thank you and really sorry this comes up now, but better now than later.

Reply:
Done